.github/workflows/main.yml

```
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    if: github.repository_owner == 'AxaFrance'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: latest
+          cache: 'npm'
+      - run: npm ci
```

```suggestion
      - run: npm run package:check
      - run: npm ci
      - run: npm run lint
```

j'ajouterai aussi le check des commit comme dans l'exemple de la doc commit lint
https://commitlint.js.org/#/guides-ci-setup

---

done

---

il manque la step de lint

---

avec l'update de la commande lint ici il faudrait mettre 
`- run: npm run lint -- --max-processes 1`

---

est-ce qu'on a besoin de lancer le lint sur la CI ?

---

Oui c'est ultra utile pour moi car encre les dev qui n'active pas l'auto fix dans leur ide ou ceux qui skip husky.

---

mais le script ne va pas faire de corrections sur le lint, donc quelle est l'idée derrière ? avoir les erreurs de lint dans la CI ?

---

c'est exactement sa, si on a mis des règles de lint en place il faut qu'elle sois respecter.
Interdit de merge du code qui ne respect pas les règles

---

on aura plus ou moins toujours des erreurs eslint ou stylelint je pense, non ? 
Par exemple actuellement on en a plusieurs centaines, ça voudrait dire bloquer des PRs

---

Alors pour moi si on met en place des linter c'est pour les utiliser, et donc le code doit respecter les linter
Sinon qu'elle est l'intérêt des linter ?